dyn_em/start_em.F

```diff
@@ -147,6 +148,8 @@ SUBROUTINE start_domain_em ( grid, allowed_to_read &
 #endif
    ALLOCATE(dz8w(IMS:IME,KMS:KME,JMS:JME),STAT=I)  ; dz8w = 0.
    ALLOCATE(z_at_q(IMS:IME,KMS:KME,JMS:JME),STAT=I)  ; z_at_q = 0.
+   ALLOCATE(PRES(IMS:IME,KMS:KME,JMS:JME),STAT=I)  ; PRES = 0.   ! ntu3m
```

Comment:
@tzuchin12 
Should there be an IF test around this allocate statement? Is it required if the NTU scheme is not run?

Reply:
These two parameters are required for the NTU scheme only.